tests/constants.py

```diff
@@ -0,0 +1,22 @@
+from enum import Enum
+
+
+class KServeDeploymentType(Enum):
+    SERVERLESS: str = "Serverless"
+    RAW_DEPLOYMENT: str = "RawDeployment"
+
+
+class ModelStoragePath(Enum):
+    FLAN_T5_SMALL: str = "flan-t5-small/flan-t5-small-caikit"
```

Comment:
```suggestion
    FLAN_T5_SMALL: str = f"flan-t5-small/flan-t5-small-{ModelFormat.CAIKIT}"
```
and move ModelFormatup